restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/sql/call-detail-records.xml

```diff
@@ -181,6 +181,11 @@
 		SELECT * FROM restcomm_call_detail_records WHERE instanceid=#{instanceid};
 	</select>
 
+  <select id="getInCompleteCallDetailRecordsByInstanceId" parameterType="string" resultType="hashmap">
+    SELECT * FROM "restcomm_call_detail_records" WHERE "instanceid"=#{instanceid} AND (UPPER("status") = ('IN_PROGRESS') OR 
```

Comment:
@anhntnguyen 
did we tested this file?
I think double quotes are not supported in this mariadb sql formats in this file.
i mean double quotes around table and column names.